zerver/lib/mention.py

```diff
@@ -46,12 +47,14 @@ def Q(self) -> Q:
 @dataclass
 class MentionText:
     text: Optional[str]
+    is_topic_wildcard: bool
     is_wildcard: bool
 
 
 @dataclass
 class PossibleMentions:
     mention_texts: Set[str]
+    message_has_topic_wildcards: bool
     message_has_wildcards: bool
```

Comment:
With ref to the other comment about `stream_wildcards`, `message_has_wildcards` here would become `message_has_stream_wildcards`, which I think reads nicely. What do you think?

Reply:
I added a prep commit to address this. (Fourth commit)